java/email-contact-form/src/Cors.java

```diff
@@ -0,0 +1,55 @@
+package io.openruntimes.java.src;
+
+import io.openruntimes.java.RuntimeContext;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+import java.lang.*;
+
+
+public class Cors {
+
+    /*
```

Comment:
I don't think this format is typical JavaDoc style

Reply:
Refactored the java comments